server/src/bootstrap/index.ts

```diff
@@ -108,6 +110,7 @@ const apiOptions = {
     }),
     authMiddleware: authMiddleware,
   },
+  enquiries: new EnquiryController(mailService, agencyService),
```

Comment:
Follow convention and supply as...
```suggestion
  enquiries: new EnquiryController({ mailService, agencyService }),
```